src/backend/base/langflow/api/v1/folders.py

```diff
@@ -89,25 +92,41 @@ def read_folders(
                 or_(Folder.user_id == current_user.id, Folder.user_id == None)  # noqa: E711
             )
         ).all()
+        folders = [folder for folder in folders if folder.name != STARTER_FOLDER_NAME]
         return sorted(folders, key=lambda x: x.name != DEFAULT_FOLDER_NAME)
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e)) from e
 
 
-@router.get("/{folder_id}", response_model=FolderReadWithFlows, status_code=200)
+@router.get("/{folder_id}", response_model=FolderWithPaginatedFlows, status_code=200)
```

Comment:
Non-backwards compatible change?

Reply:
yes!

Reply:
I understand your question, but I'm currently working on a refactor. At this point, the only established contract between frontend and backend involves API calls. If a user is retrieving flows through this endpoint, they'll need to update their application accordingly..